### ISSUE TITLE
Issue with GLIBC using LinuxBrew

### ISSUE THREAD

**Issue by lolautruche** [platformsh/homebrew-tap#2]
**OS**: Ubuntu 20.04.5
**CPU**: `Linux seneque 5.15.0-48-generic #54~20.04.1-Ubuntu SMP Thu Sep 1 16:17:26 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux`
**Homebrew version**: 3.3.7

After installing using LinuxBrew, the CLI fails with the following error message:

```
$ platform
/tmp/psh-go-8.0.23-3.82.2/php-8.0.23: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.35' not found (required by /tmp/psh-go-8.0.23-3.82.2/php-8.0.23)
/tmp/psh-go-8.0.23-3.82.2/php-8.0.23: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /tmp/psh-go-8.0.23-3.82.2/php-8.0.23)
/tmp/psh-go-8.0.23-3.82.2/php-8.0.23: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /tmp/psh-go-8.0.23-3.82.2/php-8.0.23)
2022/10/05 10:32:41 Could not execute command: could not run legacy CLI command: exit status 1
```

I have GLIBC 2.31.
```
$ ldd --version
ldd (Ubuntu GLIBC 2.31-0ubuntu9.9) 2.31
```

**Comment by lolautruche** [comment]
The same occurs using the binary directly, without using LinuxBrew

**Comment by lolautruche** [comment]
Tried by installing latest GLIBC using Homebrew, but it doesn't fix the problem.
I think this is because we compile PHP using libs installed with Homebrew.

**Comment by akalipetis** [comment]
Fixed in the latest release `4.0.0-beta.3` - we now use fully static binaries for all Linux distributions and architectures.
